edgelet/edgelet-http/src/util/hyperwrap.rs

```diff
@@ -36,17 +51,33 @@ impl Config {
         if self.null {
             Ok(Client::Null)
         } else {
-            let config = self.clone();
+            let mut builder = TlsConnector::builder();
+            if let Some(bundle) = &self.trust_bundle {
+                let certs = X509::stack_from_pem(&bundle.cert).context(ErrorKind::TrustBundle)?;
```

Comment:
These `?` also need an additional `.context(ErrorKind::Initialization)`. Eg `let certs = X509::stack_from_pem(&bundle.cert).context(ErrorKind::TrustBundle).context(ErrorKind::Initialization)?;`, just like the `ProxyConnector::from_proxy` call below.

Reply:
Thanks makes sense.